CRM/Grant/BAO/Grant.php

```diff
@@ -110,19 +110,6 @@ public static function getGrantStatusOptGroup() {
   }
 
   /**
-   * Retrieve statistics for grants.
-   *
-   *
-   * @param bool $admin
-   *
-   * @return array
-   *   Array of grant summary statistics
-   */
-  public static function getGrantStatistics($admin = FALSE) {
-    $grantStatuses = array();
-  }
```

Comment:
Yes, I should have probably made a separate commit here - but I came to a similar conclusion.  This is a placeholder function for functionality that simply doesn't exist.